skafka/src/main/scala/com/evolutiongaming/skafka/producer/Metrics.scala

```diff
@@ -0,0 +1,58 @@
+package com.evolutiongaming.skafka.producer
+
+import cats.Applicative
+import com.evolutiongaming.skafka.Topic
+import com.evolutiongaming.skafka.producer.Metrics.Latency
+
+trait Metrics[F[_]] {
+
+  def initTransactions(latency: Long): F[Unit]
+
+  val beginTransaction: F[Unit]
```

Comment:
`def` in trait please, `val` in the implementation

Reply:
Done.